framework/src/main/java/org/checkerframework/common/value/ValueTransfer.java

```diff
@@ -574,13 +574,13 @@ private AnnotationMirror createAnnotationForStringConcatenation(
         List<String> rightValues = getStringValues(rightOperand, p);
 
         boolean nullStringConcat =
```

Comment:
Please write documentation about what this means.  It's surprising that you could invert its meaning without renaming the variable.

Reply:
This was something I overlooked when I changed the names of everything related to the command-line option. Thank you for noticing it.